tests/client/ui/components/TestButton.js

```diff
@@ -7,18 +7,22 @@ import Button from 'ui/components/Button';
 describe('<Button />', () => {
```

Comment:
In case we don't already would be nice to have a test to ensure that classNames does the right thing if a duplicate Button class is passed in.